### ISSUE TITLE
Vacuum automation

### ISSUE THREAD

**Issue by norbertjoni** [HeedfulCrayon/HomeAssistantConfig#1]
Hello,

I seen your FB post about this config and it's a bit hard to follow for a beginner like me.
I would like to do similar thing but I don't need any notification about is. All I want is tag the rooms which is ready for cleaning and clean those rooms on the specific time. 
I already have a vacuum integration and the zones.
Can you help me please?

**Comment by norbertjoni** [comment]
No need :) sorry for bothering and thanks for a config

**Comment by HeedfulCrayon** [comment]
@noorbertt Sorry I missed this.  If you still need help, let me know
